packages/emitter-framework/src/typescript/components/interface-declaration.tsx

```diff
@@ -124,7 +111,7 @@ function getExtendsType(type: Model | Interface): Children | undefined {
   );
 }
 
-function membersFromType(type: Model | Interface): Children {
+function interfaceMembers({ type, children }: TypedInterfaceDeclarationProps) {
```

Comment:
interface body?

Reply:
This is a component, so capital InterfaceMembers (or InterfaceBody, sure!)

Also, don't destructure props in Alloy. You need to avoid triggering gets on props because getting a prop value that is reactive needs to happen in a reactive scope (in the future, you will get a warning about this), and destructuring destroys the signal so if either type or children change you won't get the updated values.

Reply:
its technically not used as a component though, so have to update that as well at point of usage

Reply:
Can do, and yeah it would not be obvious to future me that destructuring has this side-effect of losing reactivity so some sort of linter rule would be helpful 🙏

Reply:
Indeed, an alloy lint rule would be very nice!

Reply:
Created #6953 to address this in the future. Assigned to me, but I'll try to hold off for a few sprints 😄 

Also address all feedback so far - going to resolve any conversations where I addressed the feedback